### ISSUE TITLE
StatsBoxGrid result rendering as an H3

### ISSUE THREAD

**Issue by minimalsm** [ethereum/ethereum-org-website#4757]
**Describe the bug**

The StatsBox with charts on our homepage currently uses the H3 tag to render the result. I'm guessing this was done as a quick way to get font styling but it is poor for accessibility and SEO. 

<img width="1423" alt="Screenshot 2021-12-13 at 12 03 21" src="https://user-images.githubusercontent.com/62268199/145809092-055c1023-1c5f-4a33-9847-39f70f38d88d.png">

**To Reproduce**

1. Go to 'https://ethereum.org/'
2. Scroll down to 'The latest network statistics'
3. See error

**Expected behavior**

Result should be rendered in a span that is explicitly styled to look like an H3.

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by joneskj55** [comment]
@minimalsm I can take this one.

**Comment by minimalsm** [comment]
Thanks @joneskj55, I've assigned you to this 👌
